operator-metadata/image.yaml

```diff
@@ -27,11 +29,11 @@ labels:
   - name: "operators.operatorframework.io.bundle.channel.default.v1"
     value: "stable"
   - name: "com.redhat.delivery.operator.bundle"
-    value: "true"
+    value: "false"
   - name: "com.redhat.openshift.versions"
     value: "v4.5"
   - name: "com.redhat.delivery.backport"
-    value: "true"
+    value: "false"
```

Comment:
isn't it better just to remove all the labels which are not about the old format instead of setting them as "false"? The same for all the previous ones starting with `operators.operatorframework.io.bundle` and the openshift version one. I think they are not handled in the old format.

Reply:
Yeah you are right, them me remove them